R/calcUEdemand.R

```diff
@@ -35,14 +37,20 @@ calcUEdemand <- function() {
     group_by(across(-all_of(c("value")))) %>%
     summarise(value = mean(.data[["value"]]), .groups = "drop")
 
+  # scale demand such that average is maintained with differentiated efficiency
```

Comment:
Here it was not clear to me with respect to what you want to maintain the average. I guess it's with respect to the previous setting where the entire stock has ```bs = low``` with relative demand 1? Then you can simply write: Scale demand such that the average relative demand is 1.